app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -196,6 +196,7 @@ class LuciBuildService {
 
       final Map<String, Object> properties = target.getProperties();
       properties.putIfAbsent('git_branch', () => pullRequest.base!.ref!.replaceAll('refs/heads/', ''));
+      properties['github_build_label'] = pullRequest.labels!.any((label) => label.name == 'github_build_label');
```

Comment:
Nit: use a variable instead and add a doc explaining what this means.

Reply:
done